src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -571,10 +563,14 @@ private void adjustQuestionNumbers(int oldQuestionNumber,
      * Precondition: <br>
      * {@code newAttributes} is not {@code null}
      */
-    private void updateFeedbackQuestionWithoutResponseRateUpdate(FeedbackQuestionAttributes newAttributes)
-            throws InvalidParametersException, EntityDoesNotExistException {
-
-        updateFeedbackQuestion(newAttributes, false);
+    private void updateFeedbackQuestionWithoutResponseRateUpdate(FeedbackQuestionAttributes newAttributes) {
+        try {
+            updateFeedbackQuestion(newAttributes, false);
+        } catch (InvalidParametersException e) {
+            Assumption.fail("Invalid question.");
+        } catch (EntityDoesNotExistException e) {
+            Assumption.fail("Question disappeared.");
+        }
```

Comment:
Are there any other calls to this method other than the two above? If those two are the only calls then this should be fine, otherwise you might have to modify the other call sites to avoid redundant catch blocks.